pkg/monitortests/node/watchnodes/monitortest.go

```diff
@@ -119,3 +120,31 @@ func intervalStartDuring(needle monitorapi.Interval, haystack monitorapi.Interva
 	}
 	return false
 }
+
+func nodeDiskPressure(finalIntervals monitorapi.Intervals) []*junitapi.JUnitTestCase {
+	const testName = "[[Jira:\"Test Framework\"]] kubelet should not report DiskPressure"
```

Comment:
```suggestion
	const testName = "[Jira:\"Test Framework\"] kubelet should not report DiskPressure"
```

What's the double bracketing for?
<img width="1324" alt="image" src="https://github.com/user-attachments/assets/536b77b7-a39b-45d1-b1ee-dd137cfbad96">